youtube_dl/options.py

```diff
@@ -366,6 +366,10 @@ def _comma_separated_values_options_callback(option, opt_str, value, parser):
         '--video-password',
         dest='videopassword', metavar='PASSWORD',
         help='Video password (vimeo, smotri, youku)')
+    authentication.add_option(
+        '--password-from-keyring',
```

Comment:
Should be `--gnome-keyring` or so for symmetry with `--netrc`.

Reply:
1) libsecret is supposed to be keyring-agnostic and work with either gnome-keyring or kwallet using DBus "Secret Service", so the name should be something more generic than `--gnome-keyring`.
2) I don't get your point - how is `--netrc` "symmetric" with `--gnome-keyring`?  

What option name would you suggest?

Reply:
Option name should not contain `--password-from`. Select one that describes the mechanism in the most generic way. `--netrc` means the netrc mechanism will be used, symmetrically keyring option name should denote the keyring mechanism (or whatever it's properly called) will be used.

Reply:
Renamed to `--keyring`. Not 100% accurate, but seems more intuitive for users than `--libsecret`. https://github.com/rg3/youtube-dl/pull/14674/commits/ce0148424a15bda20fb46f67db5346a2ac836b36